NPM scripts won't run locally due to global requirement

NPM scripts have errors as references are to globals, e.g. running `npm start` brings the following:
```
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.4.3
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! mern-starter@1.0.0 start: "cross-env NODE_ENV=development nodemon index.js"
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mern-starter@1.0.0 start script "cross-env NODE_ENV=development nodemon index.js".
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the mern-starter package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env NODE_ENV=development nodemon index.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs mern-starter
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls mern-starter
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/marcusbransbury/Documents/gitRepos/mern-starter-fork/npm-debug.log
```


I'll submit a PR to fix this.

--- a/package.json
+++ b/package.json
@@ -3,16 +3,16 @@
   "version": "1.0.0",
   "description": "Boilerplate project for building Isomorphic apps using React and Redux",
   "scripts": {
-    "test": "mocha shared/tests/*.spec.js --compilers js:babel-register",
-    "test:server": "cross-env NODE_ENV=test PORT=8080 MONGO_URL=mongodb://localhost:27017/mern-test mocha --compilers js:babel-register --recursive server/tests/**/*.spec.js",
-    "start": "cross-env NODE_ENV=development nodemon index.js",
-    "start:prod": "cross-env NODE_ENV=production node index.js",
+    "test": "node_modules/.bin/mocha shared/tests/*.spec.js --compilers js:babel-register",
+    "test:server": "node_modules/.bin/cross-env NODE_ENV=test PORT=8080 MONGO_URL=mongodb://localhost:27017/mern-test mocha --compilers js:babel-register --recursive server/tests/**/*.spec.js",
+    "start": "node_modules/.bin/cross-env NODE_ENV=development node_modules/.bin/nodemon index.js",
+    "start:prod": "node_modules/.bin/cross-env NODE_ENV=production node index.js",
     "bs": "npm run clean && npm run build && npm run start:prod",
     "minify": "cleancss -o static/css/app.min.css static/css/app.css",
-    "build": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js && npm run minify",
+    "build": "node_modules/.bin/cross-env NODE_ENV=production webpack --config webpack.config.prod.js && npm run minify",
     "clean": "rimraf static/dist",
     "slate": "rimraf node_modules && npm install",
-    "lint": "eslint client server shared"
+    "lint": "node_modules/.bin/eslint client server shared"
   },
   "pre-commit": [
     "lint",
@@ -32,6 +32,7 @@
   "dependencies": {
     "babel-core": "^6.4.5",
     "body-parser": "^1.14.2",
+    "cross-env": "^1.0.7",
     "cuid": "^1.3.8",
     "express": "^4.13.4",
     "history": "^1.17.0",
@@ -60,7 +61,6 @@
     "babel-register": "^6.7.2",
     "chai": "^3.5.0",
     "clean-css": "^3.4.9",
-    "cross-env": "^1.0.7",
     "css-loader": "^0.23.1",
     "css-modules-require-hook": "^2.1.0",
     "deep-freeze": "0.0.1",

